src/terminal.rs

```diff
@@ -16,60 +19,155 @@ impl Terminal {
     }
 
     pub fn should_ask_for_todo(&mut self) -> Result<UserResponse, TerminalError> {
-        let response = self.input.read_line().map_err(TerminalError::StdinErr)?;
+        let response = self.input()?;
+
         match response.trim() {
-            "s" => Ok(UserResponse::Yes),
-            "n" => Ok(UserResponse::No),
+            "1" => Ok(UserResponse::Insert),
+            "2" => Ok(UserResponse::ShowTodos),
+            "3" => Ok(UserResponse::Update),
+            "4" => Ok(UserResponse::Delete),
+            "5" => Ok(UserResponse::Exit),
             _ => Ok(UserResponse::Other),
         }
     }
 
-    pub fn ask_for_new_todo(&mut self) -> Result<Todo, TerminalError> {
-        self.output.clear_screen().map_err(TerminalError::StdoutErr)?;
-        let ask_todo = style("\nQual TODO deseja criar?").magenta();
+    pub fn show_options(&mut self) -> Result<(), TerminalError> {
+        self.write_styled(
+            "\nEscolha uma opção para usar seu TODO LIST 🤔",
+            Style::new().magenta(),
+        )?;
+        self.write_styled("\n1 - Para CRIAR um TODO \n2 - Para LISTAR todos os seus TODO's \n3 - Para ALTERAR um TODO existente \n4 - Para DELETAR um TODO \n5 - Para SAIR\n", Style::new().white())?;
```

Comment:
Por que não usar uma string multiline aqui?

```suggestion
        self.write_styled(r"
1 - Para CRIAR um TODO
2 - Para LISTAR todos os seus TODO's
3 - Para ALTERAR um TODO existente
4 - Para DELETAR um TODO \n5 - Para SAIR", Style::new().white())?;
```

Reply:
Ficou mais legível. Obrigado!